load_postgres_data.py

```diff
@@ -0,0 +1,74 @@
+import os
```

Comment:
We should put this somewhere better than the root. I suggest `server/athenian/api/tests`

Reply:
Wherever you think is better sounds good to me 👍